src/static/build.rs

```diff
@@ -37,10 +39,22 @@ fn main() {
             cc::Build::new()
                 .file("./lib/a.c")
                 .flag("-fPIC")
+                .flag("-std=c99")
                 .flag("-lOpenCL")
                 .shared_flag(true)
                 .compile("a");
             println!("cargo:rustc-link-lib=dylib=OpenCL");
         }
+        #[cfg(target_os = "macos")]
+        {
+            cc::Build::new()
+                .file("./lib/a.c")
+                .flag("-fPIC")
+                .flag("-std=c99")
+                .flag("-framework")
+                .flag("OpenCL")
+                .shared_flag(true)
+                .compile("a");
+        }
```

Comment:
This should not be here, it was just removed in #14

Reply:
ugh sorry, i need a better diff tool :(